sklearn/impute/tests/test_impute.py

```diff
@@ -971,42 +971,39 @@ def test_iterative_imputer_catch_warning():
     assert not np.any(np.isnan(X_fill))
 
 
-@pytest.mark.parametrize("min_value, max_value",
-                         [(0, 100), (None, None), (-np.inf, np.inf),
-                          ((-5, 5, 10), (100, 200, 300)),
-                          ((-5, None, 10), (100, 200, None))])
-def test_iterative_imputer_min_max_completion(min_value, max_value):
+@pytest.mark.parametrize("min_value, max_value, correct_output",
```

Comment:
Here you can add a new line, it will be more readable for the rest of the code:

```python
@pytest.mark.parametrize(
    "min_value, max_value, correct_output",
    [(0, 100, np.array([[0] * 3, [100] * 3])),
     (...),
     ...]
)
```

Reply:
Done!